core/io/file_access.h

```diff
@@ -247,8 +294,17 @@ class FileAccess : public RefCounted {
 	static bool get_read_only_attribute(const String &p_file);
 	static Error set_read_only_attribute(const String &p_file, bool p_ro);
 
-	static void set_backup_save(bool p_enable) { backup_save = p_enable; }
-	static bool is_backup_save_enabled() { return backup_save; }
+	static void set_backup_save(bool p_enable) {
+		WARN_DEPRECATED_MSG("Use set_default_integrity_level() instead.");
+		set_default_integrity_level(p_enable ? SAVE_INTEGRITY_SAVE_SWAP : SAVE_INTEGRITY_NONE);
+	}
+	static bool is_backup_save_enabled() {
+		WARN_DEPRECATED_MSG("Use get_default_integrity_level() instead.");
+		return default_save_integrity_level > SAVE_INTEGRITY_NONE;
+	}
```

Comment:
`is_backup_save_enabled` can be removed entirely, it was only used in `FileAccess` implementation and not exposed.